### PR TITLE
Fix cleanup script and remove obsolete tests.

### PR DESCRIPTION
**Pull request by will-cromar**

### CODE REVIEW

**Review by aman2930 — APPROVED** [review]

File: tests/tensorflow/r2.4/common.libsonnet [hunk 1]
```diff
@@ -42,7 +42,7 @@ local mixins = import 'templates/mixins.libsonnet';
   },
   // Running functional tests at 10PM PST on Fri.
```
Comment (aman2930) [inline tests/tensorflow/r2.4/common.libsonnet:43]: Please update the comment also.

File: tests/tensorflow/r2.2/common.libsonnet [hunk 2]
```diff
@@ -25,7 +25,7 @@ local mixins = import 'templates/mixins.libsonnet';
   },
   // Running functional tests at 10PM PST on Thu.
```
Comment (aman2930) [inline tests/tensorflow/r2.2/common.libsonnet:26]: Please update the comment also.

**Review by aman2930 — APPROVED** [review]

File: tests/tensorflow/r2.4/common.libsonnet (hunk 1 above)
Comment (will-cromar) [reply tests/tensorflow/r2.4/common.libsonnet]: Done

File: tests/tensorflow/r2.2/common.libsonnet (hunk 2 above)
Comment (will-cromar) [reply tests/tensorflow/r2.2/common.libsonnet]: Done

**Pull request merged**